ang/crmCaseType.css

```diff
@@ -32,6 +34,14 @@
     width: 10em;
 }
 
+.crmCaseType-edit-actset {
+    border: 0 !important;
+    border-bottom: 1px solid #666 !important;
+    box-sizing: border-box;
+    float: left;
```

Comment:
why is `float: left` important?

Reply:
The icons already have `float:left`, so if this is not mentioned, the icons will appear at the left of the text box.